arc/tests/e2e/database_inspection.py

```
@@ -0,0 +1,296 @@
+        
+        # 1. CONFIGURATIONS TABLE
+        print_table_header("configurations")
+        result = await conn.execute(text("""
+            SELECT * FROM configurations 
+            ORDER BY created_at DESC 
+            LIMIT 5
+        """))
+        
+        configs = []
+        for i, row in enumerate(result, 1):
+            record = dict(zip(result.keys(), row))
```

Ruff (0.11.9)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)